### DISCUSSION TITLE
Storybook integration

### DISCUSSION THREAD

**Discussion by RyanClementsHax** [analogjs/analog#1025] [General]
Hiya 👋🏻 

What is the current status of Analog's integration with Storybook? Is the strategy to use the Angular framework integration as evidenced by https://github.com/storybookjs/storybook/issues/22544 and https://github.com/brandonroberts/analog-angular-storybook-v16 or is something Analog specific desired?

I'd be happy to help out where I can as I created https://github.com/RyanClementsHax/storybook-addon-next which later became https://github.com/storybookjs/storybook/tree/next/code/frameworks/nextjs

**Comment by brandonroberts** [comment]
👋  You can use Storybook with Analog using the Storybook Webpack builder without any special setup. 

If you want to use Vite all the way with Storybook, you'd have to set it up like the v16 repo I did and run the storybook commands directly instead of using the builders.

**Reply by RyanClementsHax** [reply]
Slick.

I take it there won't be anything special needed until analog starts exposing its own services and such.

**Comment by brandonroberts** [comment]
I opened an issue in our example repository for Storybook 7.x with Angular v17. Storybook 8.x has a different way of running Storybook with frameworks.

https://github.com/brandonroberts/angular-v17-vite-storybook/issues/7

@RyanClementsHax

**Reply by RyanClementsHax** [reply]
On it
